css/styles.css

```diff
@@ -63,3 +68,60 @@ textarea {
 .justify-end {
   justify-content: end;
 }
+
+.text-uppercase {
+  text-transform: uppercase;
+}
+
+.newBtn {
+  display: inline-block;
+}
+
+/* branch 2 */
+.flexNew {
+  display: flex !important;
+}
```

Comment:
Don't use !important. If you're needing to use this, you're probably structuring your code incorrectly